index.html

```diff
@@ -9,4 +10,355 @@
 <body>
     
 </body>
+=======
+
+<head>
+
+    <!-- Required meta tags -->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Flipcart home page</title>
+
+</head>
+
+<body>
+
+    <!--Navbar-->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top py-1">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#"><img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/f.png"
```

Comment:
@Akshay8122 Absolute path for assets like **/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/f.png** will work only on your PC. Add assets to project and replace this with relative path.

Reply:
i have fixed it